### ISSUE TITLE
Crop feature

### ISSUE THREAD

**Issue by Skylled** [photostack/photostack#125]
I finally had what I thought was a perfect use case for Photostack and checked it out today. The easiest way for me to take screenshots of GeForce Now games is to play from my phone and take Android screenshots (there's even a button on one of my controllers).

Unless your screen is exactly 16:9, this leaves black borders on the sides of the screen, which I'd love to crop out in bulk. If Photostack could do this, that could be cool. I'm imagining you'd be able to specify how much of the top, bottom, left, and right that you want to crop out.

In the example image below, I'd want to crop 142 from the left and 278 from the right (0 top, 0 bottom, obv).

![Screenshot_20210427-121112](https://user-images.githubusercontent.com/13603296/116302864-508fbf00-a767-11eb-8c6e-f67c134713c3.png)

**Comment by corbindavenport** [comment]
That would be a good feature to have, yeah. The only issue I can think of is how the crop setting would factor in with the resizing. The crop values would either have to be pre-resize or post-resize (the border setting is post-resize), I'm not sure which would be more intuative. Or maybe use percentage values?

Either way, I can look into implementing this once the transition to Bootstrap 5 is complete.

**Comment by Skylled** [comment]
I'd think pre-resize would make sense, as you'd know (or could measure) ahead of time the exact amount you want to crop in by.

Percentage would be a lot harder to work with, at least for me.

**Comment by corbindavenport** [comment]
This is live now! Gonna close this issue now, but feel free to comment (or open a new issue) if you notice something not working well.

![Screenshot 2021-06-30 at 14-48-52 PhotoStack](https://user-images.githubusercontent.com/3879063/124015507-c0623600-d9b2-11eb-9362-d16673651902.png)
